ebpfcore/ebpf_drv.c

```diff
@@ -56,26 +56,49 @@ _ebpf_driver_io_device_control(
 inline NTSTATUS
 _ebpf_result_to_ntstatus(ebpf_result_t result)
 {
+    NTSTATUS status;
```

Comment:
Introduces a new case (buffer_overflow). Just to be clear, the style guideline that you mention, is it returning from individual cases instead of break?

Reply:
Since I actually made a change in this function, and there is no styling guideline one way or the other, can I stick with the break; approach? I can lose the braces on each case if that helps reduce # lines.

Reply:
The default guidance is to not change the existing style without a reason.   I guess you can stick with it and I can still approve, but I would prefer reverting since you haven't given any technical reason to make the function be longer.   Decisions should really be made based on technical arguments and not personal preferences.

Reply:
Why is having a single return (especially for a function like this in particular) a goal?   Why not change windows_error_to_ebpf_result to be consistent with this one?   Not a big deal either way, but thanks for trying to provide a technical reason this time :)

Reply:
Also the PR description says this PR just adds two APIs.   So unrelated changes (like style changes) should not be bundled into the same PR... PRs can be merged much faster if they're decoupled than if they do multiple orthogonal things in the same one.